ldk/node/src/test.helpers.ts

```diff
@@ -0,0 +1,127 @@
+import {
+  ClientReadableStream,
+  ClientReadableStreamImpl,
+  ClientUnaryCall,
+  ClientUnaryCallImpl,
+  ServiceError,
+} from '@grpc/grpc-js/build/src/call';
+import { CallOptions, Deadline, Metadata } from '@grpc/grpc-js';
+import { ConnInfo } from './grpc/broker_pb';
+import { Session } from './grpc/session_pb';
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any
+type CallbackFunc<TResponse = any> = (
+  err: ServiceError | null,
+  response: TResponse,
+) => void;
+/**
+ * A simple callback handler that passes back the response when the callback gets invoked
+ *
+ * @param response - the response to pass through when the callback gets invoked
+ */
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any
+export function createCallbackHandler<TRequest, TResponse>(
+  response?: TResponse,
+) {
+  return (
+    request: TRequest,
+    metadata: Metadata,
+    options: Partial<CallOptions>,
+    callback: CallbackFunc,
+  ): ClientUnaryCall => {
+    const clientCall = new ClientUnaryCallImpl();
+
+    const actualCallback = callback || options || metadata;
```

Comment:
TypeScript was really busting my chops on this one. If there is a way I can define only the request + callback version and not have it gripe at me, I'm all ears

Reply:
Ah, because each of the service calls have three signatures? Sometimes the reasonable thing to do is slap `as any` on it, especially where we're working with test code and it's not worth the effort to wrangle Typescript into behaving.

Reply:
I'll take another peak. Thanks!

Reply:
`as any` did the trick. Thanks!